### PR TITLE
Updated README.md for better readability

### PR DESCRIPTION
**Pull request by NegassaB**
It's as the title says, I saw a few things in README.md that can be improved
- all emphasis heading have been changed to h4s for consistency with markdown rules,
- a few code fences were missing language specifiers, they have been added,
- a few screenshots have been moved around for better visibility

> Thanks for the work, I really appreciated it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,22 +20,27 @@ A Django application that lets users create forms. This is achieved firstly sett
...
+- Endpoints to download questions and responses both directly via views and via the admin panel.
 
 ## Installation
 
 ### PIP install and `settings.py`
+
 The application can be installed by running the following command:
-```
+
+```python
```
Comment (Salaah01) [inline README.md:38]: It'd change this to "```bash" as strictly speaking it's not a python snippet as it's something we'd run in shell.

File: README.md [hunk 2]
```diff
@@ -78,13 +88,15 @@ If you want to limit the available views, you can import each of the views direc
 This might be useful when overriding templates and defining permissions.
 
 ### Using out of the box templates
+
 The package comes with some templates to allows uses to manage and complete forms. If you wish to use these templates, you will need to download `django-crispy-forms` and update your `settings.py` file. Follow the instructions below:
 
-```
+```python
```
Comment (Salaah01) [inline README.md:94]: It'd change this to "```bash" as strictly speaking it's not a python snippet as it's something we'd run in shell.

**Review by Salaah01 — CHANGES_REQUESTED** [review]
Thank you for taking the time to update the readme, I really appreciate it!
I have left two comments, if you could revisit them I'd be happy to merge your changes.

File: README.md (hunk 2 above)
Comment (NegassaB) [reply README.md]: Idk how I missed that, resolved on 7b9462fdabdf38e926bb4963649210fa873bd570

File: README.md (hunk 1 above)
Comment (NegassaB) [reply README.md]: Idk how I missed that, resolved on 7b9462fdabdf38e926bb4963649210fa873bd570

**Comment by NegassaB** [comment]
I know this is completely off topic but I'm trying to learn testing for django and I'm interested to know how you managed it on this project

**Review by Salaah01 — APPROVED** [review]

**Pull request merged**